tests/product/test_server_uninstall.py

```diff
@@ -59,22 +57,6 @@ def assert_uninstalled_dirs_removed(self, container):
         self.assert_path_removed(container, '/usr/shared/doc/presto')
         self.assert_path_removed(container, '/etc/init.d/presto')
 
-    def test_uninstall_when_server_down(self):
```

Comment:
This can be removed because `test_uninstall_lost_host()` covers the same scenario.